contributing_to_docs/doc_guidelines.adoc

```diff
@@ -782,7 +782,7 @@ link:http://othersite.com/otherpath[friendly reference text]
 
 IMPORTANT: You must use `link:` before the start of the URL.
 
-IMPORTANT: You cannot link to a repository that is hosted on www.github.com.
+IMPORTANT: You cannot link to a repository that is hosted on www.github.com. An exception to this rule is when an autogenerated API reference contains a link to www.github.com.
```

Comment:
```suggestion
IMPORTANT: You cannot link to a repository that is hosted on www.github.com. An exception to this rule is when autogenerated API reference content contains a link to www.github.com.
```

Reply:
+1 for this change, thanks for opening the PR Sara!

Reply:
Agree. Thank you! Addressed :)